gensim/summarization/graph.py

```diff
@@ -0,0 +1,254 @@
+
+from abc import ABCMeta, abstractmethod
+
+
+class IGraph:
```

Comment:
Just a formal note, it'd be nice to keep it consistent with rest of gensim and inherit from object `class IGraph(object):`
And second thing, there are sometimes too many white spaces (based on PEP 8 style guide).